test/cpp/util/grpc_tool.cc

```diff
@@ -541,6 +541,8 @@ bool GrpcTool::CallMethod(int argc, const char** argv,
     PrintMetadata(client_metadata, "Sending client initial metadata:");
 
     CliCall call(channel, formatted_method_name, client_metadata);
+    fprintf(stderr, "New call for method_name:%s has peer address:|%s|\n",
```

Comment:
How about putting these messages behind a flag rather than logging them unconditionally?

Reply:
Done. I wasn't sure initially because we're printing these to stderr (separate from the actual RPC response payload output), but sounds reasonable.